src/main/resources/application.conf

```diff
@@ -53,29 +53,51 @@ mantis {
       # Turn discovery of/off
       discovery-enabled = true
```

Comment:
Testnet seems like a good place to see if it works. When I tried with only 1 bootstrap node and discovery disabled, I think the syncing process complained that it doesn't have enough peers to work.

Reply:
That was probably with fast sync enabled, right? If so a minimum of `min-peers-to-choose-pivot-block = 3` peers is required for it to start

Reply:
That's right. I think that could be relaxed a little bit: if you need 2 out of 3 to agree on a pivot block, than having 2 out of 2 agree should also be acceptable. I also wonder why fast-sync can only be used once in the lifetime of a node. What if it needs a restart for some reason before fast sync finishes? It would be nice if it could go back to fast-sync if it at any point sees that it is massively behind the other nodes, in which case you could further tweak the conditions for the pivot block. For example instead of a quorum of nodes having to be no more than 32 offset behind the fastest node, you could start fast syncing with whatever block everyone seems to agree on, and later adjust the target of the sync as you get closer to the tip.

Reply:
`floor(min-peers-to-choose-pivot-block / 2)+ 1`

Reply:
so:
- you can restart node ( and fast sync ) until the fast sync finish.
- when the fast sync finishes you can never fast sync again. The reason being is that fast sync is less secure that regular sync, because it does not process every transaction in the block. So even using is as one shot mechanism is a little risky, but it is a valid tradeoff to enable quick sync.. But if we would allow switching from regular sync do fast sync, every time it would happen we would expose node to more risk due to less validation being done. If you are interested the whole write up about this is here: https://github.com/ethereum/go-ethereum/pull/1889
- we are choosing pivot block to be 32 blocks behind best block in the net, to avoid any large reorgs i.e syncing to wrong branch
- you are right the quorum params probably could be tweaked, but but i do not think this has a high priority. From what the similar procedure is done in other clients, so imo for november realese what we have is good enough.